components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../shared';
+import Desktop from './sub-components/Desktop';
+import Mobile from './sub-components/Mobile';
+
+const Wrapper = styled.nav`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+`;
+
+const Pagination = ({
+  ariaLabel,
+  activePage,
+  activePageAriaLabel,
+  nextButtonText,
+  pageAriaLabel,
+  pageHref,
+  prevButtonText,
+  onPageClick,
+  totalPages,
+  infoFormatter,
+  ...props
+}) => {
+  const handlePageClick = page => e => {
+    if (!onPageClick) {
+      return undefined;
+    }
+
+    e.preventDefault();
+
+    if (page <= 0 || page > totalPages) {
+      return false;
+    }
+
+    return onPageClick(page);
+  };
+
+  const handleHref = page => {
+    if (!pageHref || onPageClick || page <= 0 || page > totalPages) {
+      return undefined;
+    }
+
+    return pageHref(page);
+  };
+
+  const width = (window && window.innerWidth) || 480;
```

Comment:
Can we use BREAKPOINTS.small instead of '480' ? <3

Reply:
Yes, of course!
Done!